tsconfig.json

```diff
@@ -30,5 +30,14 @@
     "**/*.test.ts",
     "**/FlowType*",
     "**/setup-jest.ts"
-  ]
+  ],
+  "ts-node": {
+    "transpileOnly": true,
+    "files": true,
+    "compilerOptions": {
+      "types": [
+        "node"
+      ]
+    }
+  }
```

Comment:
This should allow us to convert some more scripts over to TS, with negligible performance impact